examples/API_v2/lookup_users_using_user_ids.py

```diff
@@ -0,0 +1,19 @@
+import tweepy
+
+
+# Replace bearer token value with your own
+bearer_token = ""
+
+# Initializing the Tweepy client
+client = tweepy.Client(bearer_token)
+
+# Replace User IDs
+ids = ['2244994945']
+
+# By default the user ID, name and username are returned. user_fields can be used 
+# to specify the additional user data that you want returned for each user e.g. profile_image_url
+users = client.get_users(ids=ids, user_fields=['profile_image_url'])
```

Comment:
`ids` can be a positional argument here.

This should probably use `Client.get_user` (and not use a loop) or get multiple users.

Reply:
Added additional user id